src/util/text-tracks.js

```diff
@@ -234,7 +234,8 @@ export const addMetadata = ({
   // Map each cue group's endTime to the next group's startTime
   sortedStartTimes.forEach((startTime, idx) => {
     const cueGroup = cuesGroupedByStartTime[startTime];
-    const nextTime = Number(sortedStartTimes[idx + 1]) || videoDuration;
```

Comment:
Sorry for commenting on merged PR, but maybe we should consider `endTime`= `startTime` instead of 0 in this case?

Reply:
Good catch. Just saw a link to this from another issue. Will put up a quick PR for that fix, I agree start time is more appropriate.

Reply:
Sry to comment this, but when the endTime is set to the starttime or "0".... the cue doesn't get active anymore.